tiledb/sm/query/query_condition.cc

```diff
@@ -2575,6 +2575,10 @@ Status QueryCondition::apply_sparse(
     const ArraySchema& array_schema,
     ResultTile& result_tile,
     std::vector<BitmapType>& result_bitmap) {
+  if (!tree_) {
```

Comment:
Now that we don't allow empty conditions to be set, this check should actually be remove or throw.

Reply:
Done.